writeengine/bulk/qa-bulkload.py

```diff
@@ -182,10 +182,6 @@ def main():
   clean up old files, sort the index inserts and generally rock and roll
   """
   start_dir = curdir=os.getcwd() # remember where we started
-  
-  if not os.access('.', os.W_OK):
-    os.chdir('/tmp')
-    logger.warn('Changing to /tmp to have permission to write files')
```

Comment:
I'm not sure why this was here or why it was removed.  Does this write to the current directory?  If it does we should put this check back in these scripts.